### PR TITLE
Instrument local and remote calls separately

### PR DESCRIPTION
**Pull request by joshblum**
cc @mikem

### CODE REVIEW

File: go/client/cmd_log_send.go [hunk 1]
```diff
@@ -94,12 +94,23 @@ func (c *CmdLogSend) Run() error {
...
+	remoteNetworkStatsCmd := &CmdNetworkStats{Contextified: libkb.NewContextified(c.G()), networkSrc: keybase1.NetworkSource_REMOTE}
+	remoteNetworkStats, err := remoteNetworkStatsCmd.load()
+	if err != nil {
+		c.G().Log.Info("ignoring error getting keybase remote network stats: %s", err)
+		remoteNetworkStats = nil
+	}
+	if localNetworkStats != nil || remoteNetworkStats != nil {
+		b, err := json.Marshal(libkb.NetworkStatsJSON{
+			Local:  localNetworkStats,
+			Remote: remoteNetworkStats,
```
Comment (mlsteele) [inline go/client/cmd_log_send.go]: this does ok when it a load has failed?

**Comment by mmaxim** [comment]
You tagged the wrong person here

**Comment by mikem** [comment]
> You tagged the wrong person here

Happens occasionally ¯\\\_(ツ)\_/¯

File: go/client/cmd_log_send.go (hunk 1 above)
Comment (joshblum) [reply go/client/cmd_log_send.go]: yep

**Comment by joshblum** [comment]
oops sorry for mistag

**Pull request merged**